symplyphysics/conditions/electricity/maxwell_equations/derivative_of_magnetic_induction_in_time_is_rotor_of_electric_intensity.py

```diff
@@ -0,0 +1,30 @@
+from sympy import Eq, Derivative, Matrix
+from sympy.core import Equality
+from symplyphysics import (units, Symbol)
+from symplyphysics.core.fields.operators import curl_operator
+from symplyphysics.core.fields.vector_field import VectorField
+
+
+## Description
+## Faraday's law of induction states that a change in magnetic induction generates a vortex electric field.
+## This law is valid for any magnetic field that changes over time.
+
+## Law is: curl(E(r, t)) = -d(B(r, t)) / dt, where
+## E - the field of electric intensity,
+## B - the field of magnetic induction,
+## r - the vector of a point in space,
+## t - time,
+## curl - curl of the vector of magnetic induction,
+## d / dt - time partial derivative.
+
+time = Symbol('time', units.time)
+
+
+def faradays_law_of_induction(electric_intensity_: VectorField, 
```

Comment:
On a second thought, let's make it a law. We can have
`magnetic_induction_derivative_law(electric_intensity_: VectorField)`
and
`electric_intensity_curl_law(magnetic_induction_: VectorField)`
set of laws.